src/index.ts

```diff
@@ -14,54 +14,34 @@
  * limitations under the License.
  */
 
-/**
```

Comment:
So that's a problem, no?  We have to expose the jsdoc for the still relevant v2 version of the API.

Reply:
We do!
![image](https://user-images.githubusercontent.com/4015807/69395538-2b567180-0c94-11ea-9253-206493f76bb2.png)